build.gradle

```diff
@@ -399,7 +399,7 @@ task generateDistLinks(type: Exec, dependsOn: ['cleanDocs']) {
 task makeDocs(type: Exec, dependsOn: ['shadowJar', 'generateDistLinks']) {
     commandLine 'bash', 'python/hail/docs/makeDocs.sh'
     environment SPARK_HOME: sparkHome
-    environment PYSPARK_SUBMIT_ARGS: '--conf spark.driver.extraClassPath=' + projectDir + '/build/libs/hail-all-spark.jar --conf spark.executor.extraClassPath=' + projectDir + '/build/libs/hail-all-spark.jar pyspark-shell'
+    environment PYSPARK_SUBMIT_ARGS: '--master local --conf spark.driver.extraClassPath=' + projectDir + '/build/libs/hail-all-spark.jar --conf spark.executor.extraClassPath=' + projectDir + '/build/libs/hail-all-spark.jar pyspark-shell'
```

Comment:
This is a great idea, but let's use more resources (the notebooks have a bit of data, so more threads will help.

```
--master local[4]
``` 
perhaps?